app/assets/javascripts/timeline.js

```diff
@@ -209,6 +224,20 @@ samson.controller("TimelineCtrl", function($scope, $window, $timeout, Deploys, S
   $scope.timelineDeploys = Deploys;
   $scope.deploys = Deploys.entries;
 
+  if (!$scope.selectedTimeFormat) {
```

Comment:
can we render this into the page ?
`User.timezone = <%= current_user.timezone %>` or so ... to avoid another request ?